src/features/auth/authApiSlice.ts

```diff
@@ -0,0 +1,15 @@
+import { apiSlice } from "../api";
+
+export const authApiSlice = apiSlice.injectEndpoints({
```

Comment:
what is "`.injectEndpoints`" doing here?

Reply:
Inject endpoints is needed to add new endpoints related to authentication, could you please specify what exactly did you mean?